local SMS functional test failures

STR

1. npm run test-functional -- --grep="send_sms - valid phone number"
2. BOOM

```
AssertionError: expected 'app link sent to 916-417-1280. mistyped number?' to include '+19164171280'
 at Command.<anonymous>  <tests/functional/lib/helpers.js:1863:14>
      at Command.<anonymous>  <tests/functional/lib/helpers.js:1862:6>
      at Command.<anonymous>  <tests/functional/lib/helpers.js:57:23>
      at Test.test  <tests/functional/send_sms.js:198:10>
      at <src/lib/Test.ts:260:47>
```

This is because I wrote the tests to do formatting on the front-end, then we started to depend on the backend to format numbers. I'm not sure how these never got updated. We do not run SMS functional tests on CI because we never figured out how to make the tests run correctly.

--- a/tests/functional/send_sms.js
+++ b/tests/functional/send_sms.js
@@ -8,7 +8,6 @@ const { registerSuite } = intern.getInterface('object');
 const TestHelpers = require('../lib/helpers');
 const FunctionalHelpers = require('./lib/helpers');
 const selectors = require('./lib/selectors');
-const CountryTelephoneInfo = require('../../app/scripts/lib/country-telephone-info');
 
 const config = intern._config;
 
@@ -69,9 +68,7 @@ const suite = {
   beforeEach: function () {
     email = TestHelpers.createEmail();
     testPhoneNumber = TestHelpers.createPhoneNumber();
-    const countryInfo = CountryTelephoneInfo['US'];
-    formattedPhoneNumber =
-      countryInfo.format(countryInfo.normalize(testPhoneNumber));
+    formattedPhoneNumber = `${testPhoneNumber.substr(0, 3)}-${testPhoneNumber.substr(3, 3)}-${testPhoneNumber.substr(6)}`;
 
     // User needs a sessionToken to be able to send an SMS. Sign up,
     // no need to verify.

